fetch/main.js

```diff
@@ -0,0 +1,13 @@
+fetch('https://pokeapi.co/api/v2/pokemon/1')
```

Comment:
Please use async/await syntax instead of Promise syntax.

Other than that, it looks good.

Reply:
Adjusted code to async/await syntax!